src/styles/utilities/spacing.js

```diff
@@ -73,6 +73,10 @@ export default {
         marginRight: 20,
     },
 
+    mrn5: {
```

Comment:
Why is it called `mrn5` I see no other `mrn` in the spacing styles (same for the `mln5`) 
I think I understand it stands for `Margin Left Negative Fifth` 

Maybe leave a comment explaining that the 5 stands for a multiple of 4 (5*4=20) it's not obvious straight away

Reply:
@sketchydroide 
It's already there already said there at the top of the file,
https://github.com/Expensify/App/blob/bad9ca7a94ce30e55f77caee5c26090e48e199cd/src/styles/utilities/spacing.js#L2-L4
We don't need an extra comment to tell it's a multiple of 4.

IMO If `mln5` is understandable as `Margin Left Negative 5` then, `mrn5` is also understandable too as `Margin Right Negative 5`.

Do you still feel the need for an extra comment?

Reply:
Cool Thanks, that makes total sense.